index.html

```diff
@@ -200,53 +200,22 @@ <h1>About me</h1>
                         </ul>
                     </li>
                 </ul>
-                <!--<div class="tech">
-                    <div class="openable">
-                        <p>Languages</p>
-                        <img src="images/Techs/icons8-arrow-down-50.png">
-                    </div>
-                    <div class="techflex">
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-javascript-48.png">
-                            <p>JavaScript</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-html5-48.png">
-                            <p>HTML</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-css3-48.png">
-                            <p>CSS</p>
-                        </div>
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Frameworks</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Skills</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">-->
+
```

Comment:
You did a great description of your PR, however, you are missing a title. Please make sure to always add a title to your pull request. In addition, I think it's better practice to do your PR description using bullet points instead of long paragraphs. For example, you can do something like this:

# Added contact form and finished mobile version

For this milestone, the following was added:

- Contact form using formspree to store and receive the user's messages
- Flexbox to center input elements
- etc...


Kindly add a title to this PR. Also, make sure to credit your coding partner in the author section of your README:

![authors](https://github.com/alexansaa/Personal_Portafolio/assets/111214189/c9fa986b-f3b8-4c57-9e97-3c97cefd0fb8)